networking/network_security/AdminNetworkPolicy/ovn-k-anp-banp-metrics.adoc

```diff
@@ -0,0 +1,11 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ovn-k-anp-banp-metrics"]
+= Monitoring and managing ANP and BANP
```

Comment:
do you feel anything in this doc suggests managing as the right word?
I'd vote Monitoring ANP and BANP followed by "Metrics" sub heading, but feel free to say nay here.

Reply:
Well, no not anymore because there aren't any instructions on looking at policy.

Reply:
ack yeah sorry I can't remember what the previous version looked like.. I have a bad memory :(

Reply:
no worries.